Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/StereoPicture/EyeLink_StereoPicture.m

```diff
@@ -0,0 +1,408 @@
+function EyeLink_StereoPicture(stereoMode, screenNumber)
+% EyeLink integration demo for stereo presentation.
+% Records eye movements passively while presenting a stereo stimulus. Supports both split-screen mode 
+% and dual-monitor setup.
+% Each trial ends when the space bar is pressed.
+% Data Viewer integration with both left and right eyes superimposed on the same eye window view
+%
+% Usage:
+% Eyelink_StereoPicture(stereoMode, screenNumber)
+%
+% ------------------------------------------------------------
+% Supported stereoMode parameters:

```

Comment:
The approach from ImagingStereoDemo.m allows for way more stereo modes and elegance!

Reply:
understood and agreed the StereoMode demo needs to be revamped to reflect the most current PTB conventions. I will include this as a priority item in the next round of revisions following completion of the rest of this pull request to allow for additional time to make those changes and thorough testing before publishing